BondageClub/Assets/Female3DCG/Female3DCG.js

```diff
@@ -1535,6 +1541,12 @@ var AssetFemale3DCG = [
 					{ Name: "Shine", AllowColorize: false}
 				]
 			},
+			{ Name: "DildocornHorn", Value: -1, DefaultColor: "#Default", BuyGroup: "UnicornHorn", BodyCosplay: true, Attribute: ["PenetrateItem"], Top: -30,
```

Comment:
Same comment regarding `DefaultColor` here.
```suggestion
			{ Name: "DildocornHorn", Value: -1, DefaultColor: "Default", BuyGroup: "UnicornHorn", BodyCosplay: true, Attribute: ["PenetrateItem"], Top: -30,
```

Reply:
Absolutely, it must have slipped by when replacing the code with the default. Already fixed